boards/kernel_layout.ld

```diff
@@ -86,8 +86,9 @@ SECTIONS
         BYTE(0x6e) /* n */
         BYTE(0x65) /* e */
         BYTE(0x6c) /* l */
-        /* Kernel Major Version: 0 */
-        BYTE(0x00)
+        /* Start attributes on next page */
+        FILL(0x00)
+        . = _attribute_table_start + 512;
```

Comment:
Why do attributes need to be on a fresh page?

Reply:
They don't have to be, but it's to match how the bootloader works. The bootloader is that way because:

1. It leaves space for additional defined properties (the bootloader has a version string there).
2. It makes the 64 byte attributes nicely aligned.

It would also be beneficial to mirror the bootloader because the Tockloader code can be reused.

Reply:
I don't think we need to blank out a whole page to do this, this was set up to mirror the bootloader originally, it just does it slightly differently (writing bytes in the linker script instead of writing them in the source file, c.f. https://github.com/helena-project/tock-bootloader/blob/master/src/attributes.c#L29) -- we can adapt this to match the bootloader alignment requirements without burning a whole page.

Reply:
(resolved out-of-band, I'd missed https://github.com/helena-project/tock-bootloader/blob/master/src/attributes.c#L20, this page alignment *does* make things match the bootloader in practice; no objections left here)